### PR TITLE
Week 2 step 2

### PR DESCRIPTION
**Pull request by michaellawes**
Add fetch() functionality to page with button and javascript function in script.js to dynamically server HTML

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -32,6 +32,11 @@ <h2>Photo Gallery</h2>
       <button onclick="addRandomGreeting()">Output Fact/Quote</button>
       <div id="greeting-container"></div>
       <p>Congratulations you made it to the end of my portfolio :) I'd say it looks pretty good for someone with 4 midterms this week, 3 of them being on Thursday.</p>
+      <p>Week-2 Button</p>
+      <button onclick="showWeek2();">
```
Comment (andiov) [inline portfolio/src/main/webapp/index.html]: You don't need ;

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -32,6 +32,11 @@ <h2>Photo Gallery</h2>
...
+	      Get Text
+      </button>
+      <p id="week-2-container"></p>
```
Comment (andiov) [inline portfolio/src/main/webapp/index.html]: In general, try and keep the id names related to the project itself not weeks.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +26,11 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function showWeek2() {
+	  const responseFromServer = await fetch('/week-2');
+	  const textFromResponse = await responseFromServer.text();
+
+	  const week2Container = document.getElementById('week-2-container');
+	  week2Container.innerText = textFromResponse;
```
Comment (sarahrittgers26) [inline portfolio/src/main/webapp/script.js]: Can you fix the indentation?

**Review by sarahrittgers26 — CHANGES_REQUESTED** [review]

**Review by michaellawes — COMMENTED** [review]
I made the requested changes to the files, should I make a new branch and submit another pull request?

**Review by andiov — APPROVED** [review]

**Comment by andiov** [comment]
In general, you want to reply to each comment on your change to say whether you've addressed the issue or why not. Once that happens, we review and approve or leave more comments. You do not need to create a new PR.

**Comment by michaellawes** [comment]
Ok understood thank you   From: andiovSent: Wednesday, March 3, 2021 12:10 PMTo: michaellawes/software-product-sprintCc: michaellawes; AuthorSubject: Re: [michaellawes/software-product-sprint] Week 2 step 2 (#5) In general, you want to reply to each comment on your change to say whether you've addressed the issue or why not. Once that happens, we review and approve or leave more comments. You do not need to create a new PR.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Review by michaellawes — COMMENTED** [review]
Completed week 2 step 2

**Review by sarahrittgers26 — APPROVED** [review]

**Pull request merged**